### PR TITLE
fix: fix twitter links

### PR DESCRIPTION
**Pull request by lynnshaoyu**
links to NATIVE chains were also broken (ie: polygon, ethereum). fixed those too in this pr. 

https://uniswaplabs.atlassian.net/browse/WEB-1528?atlOrigin=eyJpIjoiZDk3YzdiNTc3NDFkNDZjYzlmNzEyMjcxYjU4NTYxMTciLCJwIjoiaiJ9

### CODE REVIEW

File: src/components/Tokens/TokenDetails/ChartSection.tsx [hunk 1]
```diff
@@ -120,7 +120,13 @@ export default function ChartSection({
         </TokenNameCell>
         <TokenActions>
           {token.name && token.symbol && token.address && (
-            <ShareButton tokenName={token.name} tokenSymbol={token.symbol} tokenAddress={token.address} />
+            <ShareButton
+              tokenName={token.name}
```
Comment (vm) [inline src/components/Tokens/TokenDetails/ChartSection.tsx:124]: should we just pass in the token?

**Review by vm — APPROVED** [review]

**Review by JFrankfurt — APPROVED** [review]
nit: would prefer no logic inside the template string

File: src/components/Tokens/TokenDetails/ChartSection.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/ChartSection.tsx]: done

**Comment by lynnshaoyu** [comment]
> nit: would prefer no logic inside the template string

done

**Pull request merged**